lib/config.js

```diff
@@ -161,6 +168,16 @@ function normalizeNumbers (opts) {
   })
 }
 
+function maybeSplit (value) {
+  return typeof value === 'string' ? value.split(',') : value
```

Comment:
Nice! I was thinking that we needed to support this from the command line as well. I like this pragmatic approach 👍

Reply:
Does this handle spaces after the comma?

Reply:
No, but I don't think that's necessary as long as it's documented